### DISCUSSION TITLE
Homework

### DISCUSSION THREAD

**Discussion by ciberado** [ciberado/azure-intro-ed01#2] [sesión 01]
Por favor, encontrar tiempo para:

* Terminar los labs que nos quedaron pendientes.
* Repasar los recursos que habéis creado utilizando el portal.

Si os veis fuertes:

* Repetid el proceso con la parte frontal de Real World, enlazando ambas aplicaciones.

**Comment by xavier-ramentol** [comment]
hola, estamos ejecutando el comando del fichero 150-app-service.md.

az webapp deploy \
  --resource-group $MYPREFIX-rg \
  --name $MYPREFIX-app \
  --src-url "https://${MYPREFIX}repositorysa.blob.core.windows.net/appversions/app.zip" \
  --type 'zip' \
  --track-status

pero nos sale un bad request....

![image](https://github.com/user-attachments/assets/120c02af-dd72-49c5-bd49-a14991801d35)

a que puede ser debido? muchas gracias.

**Reply by ciberado** [reply]
Umh... prueba a utilizar el otro método, basado en SAS (From a Storage Account using SAS) y dime si también te dice bad requet. Dejo la issue abierta y en cuanto tenga un ratico reviso el comando. Gracias!

**Reply by xavier-ramentol** [reply]
hola Javi, desde el SAS no daba problema.
